pulsar-client/src/main/java/org/apache/pulsar/client/impl/schema/KeyValueSchema.java

```diff
@@ -53,6 +53,8 @@
     @Getter
     private final KeyValueEncodingType keyValueEncodingType;
 
+    protected SchemaInfoProvider schemaInfoProvider;
```

Comment:
don't you need to provide a default implementation which returns the latest version of key/value schema?